README.md

```diff
@@ -1 +1,49 @@
-# ruby-group-project
\ No newline at end of file
+# Catalog of my things
+> A command-line-based Catalog of my things, application built with object-oriented programming (OOP). functionalities include adding and dsplaying books, music albums, games etc.
+
+ ***
+> [App Demo](https://drive.google.com/file/d/1jEc1Fhx5qG0CGfsM5AGAJcoYys7MPBlG/view?usp=sharing)
+## Built With
+> Ruby
+***
+## Getting Started
+
+* Clone the repository to get a local copy
+* From commandline of your preferred ruby enviroment navigate to the file using cd <file path of the cloned repository>
+* Run the app using ruby main.rb  or require main.rb
+* Click on the numbers representing your options and follow the command prompt
+***
+### Prerequisites
+> Ruby
+***
+### Author
```

Comment:
- [ ] OPTIONAL
I actually think you should change this sub-heading from `Author` to `Authors`, as we have more than one contributor to this project. Doing so will make your work more professional